CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ Note that this project **does not** adhere to [Semantic Versioning](https://semv
 
 ### Changed
 
+- We changed the Cleanup that handled MSC code conversion into a Formatter [#13233](https://github.com/JabRef/jabref/pull/13233)
```

Comment:
Can be removed - no change since the **last** release

Modify the other changelog entry (if needed) - I didn't check